File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
//Ask user for height
{
    int height = get_int("Height: ");

    while (height < 1 || height >= 9)
    {
        height = get_int("Height: ");
    }

    for (int i = 0; i < height; i++)
    {
        for (int d = i; d < height - 1; d++) //print dots before hashes
        {
            printf(" ");
        }

        for (int r = 0; r < i + 1; r++) //print every repetition of hashes in a new line
        {
            printf("#");
        }
        for (int d = 0; d < 2; d++) //print dots before hashes
        {
            printf(" ");
        }

        for (int r = 0; r < i + 1; r++) //print every repetition of hashes in a new line
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario/more/mario.py
from cs50 import get_int
def main():
    #ask user for height
    height = get_int("Height: ")

    hash = "#"
    dot = " "

    while (height < 1 or height >= 9):
        height = get_int("Height: ")


    for i in range(1, height + 1):

        #print dots before hashes
        for d in range(height - i):
            print(dot, end="")


        #print every repetition of hashes in a new line
        for r in range(i):
            print(hash, end="")
            r += 1

        #print dots before hashes
        for d in range(2):
            print(dot, end="")
            d += 1

        #print every repetition of hashes in a new line
        for r in range(i):
            print(hash, end="")
            r += 1

        print("\n", end="")
        i += 1


if __name__ == '__main__':
    main()